src/index.css

```diff
@@ -11,3 +13,157 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+.header {
+  text-align: center;
+}
+
+.title {
+  background: linear-gradient(to right, violet, orange, green, cyan, blue);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-family: "Young Serif", serif;
+  font-size: 50px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.question {
+  color: rgb(79, 79, 79);
+}
+.inputContainer {
+  display: flex;
+  align-items: left;
+  flex-direction: column;
+  text-align: left;
+  background: rgb(232, 231, 231);
+  border: 1px solid;
+  border-color: black;
+  padding: 10px 0 30px 30px;
+  margin-bottom: 20px;
+  width: 800px;
+  box-shadow: 11px 7px 0px 2px #000000;
+}
+
+.input {
+  padding: 10px;
+  width: 90%;
+}
+
+.textBox {
+  border: 1px solid;
+  padding: 20px;
+  margin: 20px 0 20px 0;
+  width: 750px;
+  box-shadow: 11px 7px 0px 2px #000000;
+}
+
+.likeBtn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
```

Comment:
Looks like the number of likes shrinks when the screen is 665px or smaller. Could you pls doublecheck that?